vtex-apps/search-result
# Update default 'orderBy' value to be an empty string

#### What problem is this solving?

This enables search results to be displayed in the way they were sorted inside a collection. Currently, this is not possible.

#### How to test it?

1. Go to this [Workspace](https://cosmetics1.myvtex.com/6008?map=productClusterIds), where there's a collection page. Notice how the products are ordered.
2. Check this same page in the [Portal](https://cosmetics1.vtexcommercestable.com.br/6008?map=productClusterIds). Notice how the products are **not** displayed in the same order. This is the **correct** ordering, as these products were ordered using the new Collections admin.
3. Now go to this [workspace](https://victormiranda--cosmetics1.myvtex.com/6008?map=productClusterIds), where this change is linked. Now the products should be ordered the same as in the Portal store.

#### Screenshots or example usage:

<img width="252" alt="Screen Shot 2020-10-28 at 16 32 11" src="https://user-images.githubusercontent.com/27777263/97487090-26d21880-193b-11eb-9e5d-113f541372a9.png">

#### How does this PR make you feel? [:link:](http://giphy.com/)



![sorting](https://media.giphy.com/media/oWQzTz2A4fp1m/giphy.gif)
